README.md

```diff
@@ -1,43 +1,54 @@
 # Missing tests for Laravel's auth module
+
 [![Latest Stable Version](https://poser.pugx.org/dczajkowski/auth-tests/version)](https://packagist.org/packages/dczajkowski/auth-tests)
 [![License MIT](https://poser.pugx.org/dczajkowski/auth-tests/license)](https://packagist.org/packages/dczajkowski/auth-tests)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
 
 ![](https://i.imgur.com/1z5XkDc.png)
 
 ## Versioning
+
 The version of this package reflects current major version of the Laravel framework. For example:
 If Laravel framework has version 5.6, version of this package compatible will be `5.6.*`.
 
 ## Installation
+
 ```bash
 composer require dczajkowski/auth-tests --dev
-php artisan make:auth # if not ran previously
```

Comment:
I think this is important to let people know running `make:auth` is required for tests to pass.

Reply:
The `make:auth` command was removed in Laravel 6.
Now the initial UI scaffolding can be done by installing `laravel/ui` package and running `php artisan ui bootstrap --auth` or php artisan ui vue --auth` and running `npm install && npm run dev`

Reply:
Oh, I did not know that. Haven't had time to look at 6.0 release notes at all. Thanks for the heads up! I will look into that in the upcoming days.